zebra/rib.h

```diff
@@ -481,6 +481,7 @@ int zebra_rib_queue_evpn_rem_vtep_del(vrf_id_t vrf_id, vni_t vni,
 				      struct in_addr vtep_ip);
 
 extern void meta_queue_free(struct meta_queue *mq);
+extern void meta_queue_free_vrf(struct meta_queue *mq, vrf_id_t vrf_id);
```

Comment:
can we maybe rename these to be "rib_meta_queue_xxx" ?

Reply:
Done